ocs_ci/helpers/helpers.py

```diff
@@ -3926,3 +3926,28 @@ def get_noobaa_db_used_space():
         f"noobaa_db used space is {df_out[-4]} which is {df_out[-2]} of the total PVC size"
     )
     return df_out[-4]
+
+
+def clean_all_test_projects(project_name="test"):
+    """
+    Delete all namespaces with 'test' in its name
+    'test' can be replace with other string
+
+    Args:
+        project_name (str, optional): expression to be deleted. Defaults to "test".
+
+    """
+    oc_obj = OCP()
+    all_ns = oc_obj.exec_oc_cmd(
+        "get ns -ocustom-columns=NAME:.metadata.name", out_yaml_format=False
```

Comment:
Why you don't prefer the yaml format? You can easily filter whatever data out of it using filter function:
https://thispointer.com/python-filter-a-dictionary-by-conditions-on-keys-or-values/#:~:text=Filter%20a%20Dictionary%20by%20keys%20in%20Python%20using%20filter()